cmd/root.go

```diff
@@ -0,0 +1,58 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	t "github.com/AirHelp/filler/templates"
+	"github.com/spf13/cobra"
+)
+
+const (
+	directoryToScanArgument = "dir"
+	fileExtensionArgument   = "ext"
+)
+
+var rootCmd = &cobra.Command{
+	Use:   "filler",
+	Short: "Filler - fill templates with environment variables",
+	Long:  "Filler - fill templates with environment variables",
+	RunE:  template,
+}
+
+func Execute() {
+	if err := rootCmd.Execute(); err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+}
+
+func init() {
+	rootCmd.PersistentFlags().String(directoryToScanArgument, "", "directory where app will search for templates")
+	rootCmd.PersistentFlags().String(fileExtensionArgument, "", "template file extension")
+}
+
+func template(cmd *cobra.Command, args []string) error {
+
+	dirToScan, err := cmd.Flags().GetString(directoryToScanArgument)
+	if err != nil {
+		return err
+	}
+	if dirToScan == "" {
+		return errors.New("directory path is missing")
+	}
```

Comment:
can we also check if given directory exists?
```
$▶ ./filler --dir ble/ble/ble --ext tpl; echo $?
0
```

Reply:
https://github.com/AirHelp/filler/blob/d9a26b35dd4612e820dbff61cb1c4cfcdb890189/templates/templates.go#L86